browser/ui/webui/settings/brave_sync_handler.cc

```diff
@@ -99,7 +115,17 @@ void BraveSyncHandler::HandleGetQRCode(const base::ListValue* args) {
   CHECK(args->Get(0, &callback_id));
   const base::Value* sync_code;
   CHECK(args->Get(1, &sync_code));
-  // TODO(petemill): Wait until at least cr82 for QRCodeGenerator to be in tree.
+
+  std::vector<uint8_t> seed;
+  if (!brave_sync::crypto::PassphraseToBytes32(sync_code->GetString(), &seed)) {
+    LOG(ERROR) << "invalid sync code:" << sync_code->GetString();
+    ResolveJavascriptCallback(*callback_id, base::Value(false));
+    return;
+  }
+
+  std::string seed_serialized = StrFromUint8Array(seed);
```

Comment:
Why do we need this? This is not required for `QRCodeGenerator::Generate`

Reply:
I will need `StrFromUint8Array` to send the output of QRCode::Generate, if I'm rendering it in JS 😞